rest/route/sns.go

```diff
@@ -57,45 +56,37 @@ func makeBaseSNS(env evergreen.Environment, queue amboy.Queue) baseSNS {
 	}
 }
 
-func (sns *baseSNS) Parse(ctx context.Context, r *http.Request) error {
-	sns.messageType = r.Header.Get("x-amz-sns-message-type")
-
-	body, err := io.ReadAll(r.Body)
-	if err != nil {
-		return errors.Wrap(err, " reading body")
-	}
-	if err = json.Unmarshal(body, &sns.payload); err != nil {
-		return errors.Wrap(err, "unmarshalling JSON payload")
+func (bsns *baseSNS) Parse(ctx context.Context, r *http.Request) error {
+	bsns.messageType = r.Header.Get("x-amz-sns-message-type")
+	payload := getSNSPayload(r.Context())
+	if (payload == sns.Payload{}) {
```

Comment:
The parentheses are redundant and can be removed.

Reply:
I would think so too, but it doesn't compile without 
`syntax error: unexpected { at end of statement`